admin/fleetcommander/utils.py

```diff
@@ -115,3 +116,15 @@ def merge_settings(a, b):
         result[domain] = [index[key] for key in index]
 
     return result
+
+
+def get_free_port():
```

Comment:
Strictly speaking, isn't this the same as using -1 as the port number or is that something that Soup prevents us from doing?